### DISCUSSION TITLE
Create a Wear OS version for the app

### DISCUSSION THREAD

**Discussion by fnfontana** [iSoron/uhabits#1507] [Feature Requests]
First, I need to congratulate all the people involved in the project because it's exceptional, there's about 2 years or so that I use this app, and it's getting better. I have tested a lot of other similar apps, but none of then deliver such complete experience. And for free!

Now, talking about this feature request. It is a well known concept in the cognitive sciences that than fewer obstacles do we have between our habits and routines, the easier is to attach to it and more likely to maintain it.

Giving one of my latest experience as an example, a couple of months ago I used to have much difficult to maintain me hydrated. I have tried to use alarms, reminders, some specific apps to remind me to drink water, but was not working. Until I have bough a smartwatch.

I have installed an app into my watch called [waterminder](https://waterminder.com/), and at the end of the day I register that I have used this app at the habit tracker. 

I think that what works for me was having the app very easy to access right on my wrist. This didn't happen to my phone, because I deliberately keep it far from me when at work. 

So this gave me the idea to search if Loop Habit Tracker also have a Wear OS version, but unfortunately I didn't find it. 
The concept that I have in mind is to have a simple and efficient app, with a clean interface. Just like this app already is on the phone!. The Wear OS app that I have mentioned before (waterminder) is another good reference. The smartwatch app will be synchronized to the phone app and, if possible, it could work as a standalone as well.

_The main goal for this idea is to keep the habit of tracking another habits the more ease and fluid as possible, so we cannot forget or give an excuse to ourselves to not track our habits and progress._

I would like to suggest for the development community to consider this possibility. Also, I would like to say that I know how to code, but I'm not experienced with Android or Android Wear apps, but if there is something that I could contribute, I will appreciate it.

**Comment by anivs** [comment]
来信已收到(*^_^*) ~~

**Comment by clemenceroumy** [comment]
That would be very nice ! Maybe something as the Android widget would be enough

**Comment by samyuh** [comment]
I can work on this if an issue is open :D

**Reply by br34chy** [reply]
I second!

**Comment by free4fun** [comment]
I have open the issue [1977](https://github.com/iSoron/uhabits/issues/1977)
Thank you!

**Comment by br34chy** [comment]
Is there an update with this? Has anyone started?

**Comment by thehale** [comment]
This is definitely a workaround, but I made an app called [Habit Sync for Todoist](https://github.com/thehale/HabitSync-for-Todoist), which can mark off habits when you complete a task in Todoist.

Since Todoist has a WearOS app, when I complete tasks from my wrist they get synced over to the Loop Habit app.

**Comment by f5b7** [comment]
There is an old discussion about this:
https://github.com/iSoron/uhabits/discussions/82

Maybe https://github.com/Querschlag or https://github.com/matthewfcarlson can share their experience with what they coded and discussed with @iSoron and what the obstacles are/were?

Or some Wear OS developer in this discussion might give it a(nother) try, @samyuh or @clemenceroumy for example? 🥺👉👈🤗

**Reply by clemenceroumy** [reply]
I *might* give it a try (although not 100% sure, because i already work on my own stuff)
